### PR TITLE
feat: add sitemap

### PR DESCRIPTION
**Pull request by fairlighteth**
- Adds a sitemap using the https://www.npmjs.com/package/next-sitemap package
- Is generated post build `yarn build`

Screenshot 2023-07-06 at 18 59 10

Screenshot 2023-07-06 at 18 58 35

### CODE REVIEW

File: lingui.config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  locales: ['en', 'es', 'pseudo'],
```
Comment (fairlighteth) [inline lingui.config.js:2]: Had to remove, otherwise it would add links like cow.fi/es/tokens....

Comment (anxolin) [reply lingui.config.js:2]: Ok.... for now, until @avsavsavs or I translate into 🇪🇸

File: public/sitemap-0.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+<url><loc>https://cow.fi/tokens/manifold-finance</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/matic-network</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/metaverse-index</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/origin-ether</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/reflexer-ungovernance-token</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/ribbon-finance</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/rocket-pool</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+</urlset>
```
Comment (anxolin) [inline public/sitemap-0.xml:47]: do we need to commmit this file in the project? if not we should delete and add to .gitignore

File: public/sitemap.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+<sitemap><loc>https://cow.fi/sitemap-0.xml</loc></sitemap>
```
Comment (anxolin) [inline public/sitemap.xml:3]: same, are you sure we need to add it?

**Review by anxolin — COMMENTED** [review]
Cool, we can do a hotfix with this!

File: public/sitemap-0.xml (hunk 2 above)
Comment (fairlighteth) [reply public/sitemap-0.xml:47]: Thought about it too. In principle  it is dynamic of nature so therefore I would indeed to gitignore

File: lingui.config.js (hunk 1 above)
Comment (fairlighteth) [reply lingui.config.js:2]: ¡Vale, perfecto!

File: public/sitemap.xml (hunk 3 above)
Comment (fairlighteth) [reply public/sitemap.xml]: Removed now with updated gitignore

**Pull request merged**